### PR TITLE
Ctest: Adding tests for ascicgpu031

### PR DESCRIPTION
**Pull request by csiefer2**
Standing up no-uvm tests

### CODE REVIEW

**Review by kddevin — DISMISSED** [review]

File: cmake/ctest/drivers/ascicgpu031/setSshEnv.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+# private key.  To use this in bash:
+#
+#     eval $(python ./setSetSshEnv.python)
+#
+# It assumes that ssh creates files of the form /tmp/ssh-Abcdefg12345/agent.12345 .
+#
+# TODO 1: It would be better to skip the socket query and instead look directly for the ssh-agent lock files.
+
+# Fingerprint of identity that you will use. You can find it with "ssh-add -l".
+keyFingerprint = "4096 db:7b:ce:13:b4:88:64:59:b2:e9:b8:17:f0:a6:97:d5 id_rsa_geminga (RSA)"
```
Comment (jhux2) [inline cmake/ctest/drivers/ascicgpu031/setSshEnv.py:18]: Who's gonna own the agent/SSH key?

**Review by jhux2 — COMMENTED** [review]
@csiefer2 RECIPIENTS should be tpetra-regression, no?

File: cmake/ctest/drivers/ascicgpu031/sendTestSummary.sh [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+USER=`whoami`
+
+#who gets the email summary
+if [[ $DEBUGMODE == 1 ]]; then
+  RECIPIENTS=(
+    "${USER}@sandia.gov"
+  )
+else
+  RECIPIENTS=(
+    "muelu-regression@software.sandia.gov"
```
Comment (jhux2) [inline cmake/ctest/drivers/ascicgpu031/sendTestSummary.sh:54]: This should be `tpetra-regression`.

**Review by jhux2 — CHANGES_REQUESTED** [review]

**Review by jhux2 — APPROVED** [review]
lgtm

**Pull request merged**